spec/helpers/locations_spec.rb

```diff
@@ -74,6 +74,45 @@
     end
   end
 
+  context 'when using Alma' do
+    before do
+      allow(Rails.configuration).to receive(:use_alma).and_return(true)
+    end
+
+    let(:fallback) { 'Fallback' }
+    let(:without_code) { { 'library' => 'Library Name', 'location' => fallback } }
+    let(:invalid_code) { { 'library' => 'Library Name', 'location' => fallback, 'location_code' => 'invalid' } }
+    let(:valid_code) { { 'library' => 'Library Name', 'location' => fallback, 'location_code' => 'aas' } }
+
+    it 'returns holding location label when location code lookup successful' do
+      stub_holding_locations
```

Comment:
Initially I attempted to use a new file I added with the alma holding locations. I ended up removing it because I was getting a circleci error and could not figure it out. https://app.circleci.com/pipelines/github/pulibrary/orangelight/1290/workflows/817f960c-6a98-49b8-8871-4174742b5b9d/jobs/2569

Reply:
You mean all the location stubs are in one fixture file?